2021-05-28 15:49:49,675 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-28 15:50:13,537 ERROR Exception on /clients [GET]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    return self.finalize_request(rv)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1535, in finalize_request
    response = self.make_response(rv)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1698, in make_response
    raise TypeError(
TypeError: The view function for 'show_all_clients' did not return a valid response. The function either returned None or ended without a return statement.
2021-05-28 15:50:13,537 INFO 127.0.0.1 - - [28/May/2021 15:50:13] "[35m[1mGET /clients HTTP/1.1[0m" 500 -
2021-05-28 15:51:43,067 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-28 15:51:50,281 INFO 127.0.0.1 - - [28/May/2021 15:51:50] "[33mGET / HTTP/1.1[0m" 404 -
2021-05-28 15:51:54,668 ERROR Exception on /clients [GET]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    return self.finalize_request(rv)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1535, in finalize_request
    response = self.make_response(rv)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1698, in make_response
    raise TypeError(
TypeError: The view function for 'show_all_clients' did not return a valid response. The function either returned None or ended without a return statement.
2021-05-28 15:51:54,668 INFO 127.0.0.1 - - [28/May/2021 15:51:54] "[35m[1mGET /clients HTTP/1.1[0m" 500 -
2021-05-28 15:52:30,467 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-28 15:52:34,732 INFO 127.0.0.1 - - [28/May/2021 15:52:34] "GET /clients HTTP/1.1" 200 -
2021-05-30 12:58:02,708 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:00:08,516 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:07:52,698 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 25, in create_client
    client = Client(body["clientId"], body["clientName"])
KeyError: 'clientId'
2021-05-30 13:07:52,725 INFO 127.0.0.1 - - [30/May/2021 13:07:52] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:09:37,938 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 26, in create_client
    client_service.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 14, in create_client
    return self.client_dao.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 12, in create_client
    cursor.execute(sql, (client.client_name))
TypeError: not all arguments converted during string formatting
2021-05-30 13:09:37,940 INFO 127.0.0.1 - - [30/May/2021 13:09:37] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:11:43,119 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:11:48,648 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 26, in create_client
    client_service.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 14, in create_client
    return self.client_dao.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 13, in create_client
    cursor.execute(sql, (client_name))
TypeError: not all arguments converted during string formatting
2021-05-30 13:11:48,652 INFO 127.0.0.1 - - [30/May/2021 13:11:48] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:13:15,380 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:13:22,361 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 26, in create_client
    client_service.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 14, in create_client
    return self.client_dao.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 13, in create_client
    cursor.execute(sql, client_name)
TypeError: not all arguments converted during string formatting
2021-05-30 13:13:22,364 INFO 127.0.0.1 - - [30/May/2021 13:13:22] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:15:13,637 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:15:16,114 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 26, in create_client
    client_service.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 14, in create_client
    return self.client_dao.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 13, in create_client
    cursor.execute(sql)
psycopg2.errors.SyntaxError: syntax error at or near "%"
LINE 1: insert into client (client_name) values (%s) returning clien...
                                                 ^

2021-05-30 13:15:16,118 INFO 127.0.0.1 - - [30/May/2021 13:15:16] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:16:22,386 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:16:30,746 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 26, in create_client
    client_service.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 14, in create_client
    return self.client_dao.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 13, in create_client
    cursor.execute(sql, "Test")
TypeError: not all arguments converted during string formatting
2021-05-30 13:16:30,750 INFO 127.0.0.1 - - [30/May/2021 13:16:30] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:17:25,535 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:17:28,390 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 26, in create_client
    client_service.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 14, in create_client
    return self.client_dao.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 13, in create_client
    cursor.execute(sql, (client.client_id, client.client_name))
TypeError: not all arguments converted during string formatting
2021-05-30 13:17:28,393 INFO 127.0.0.1 - - [30/May/2021 13:17:28] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:18:30,813 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:18:32,956 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 26, in create_client
    client_service.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 14, in create_client
    return self.client_dao.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 13, in create_client
    cursor.execute(sql, str(client.client_name))
TypeError: not all arguments converted during string formatting
2021-05-30 13:18:32,960 INFO 127.0.0.1 - - [30/May/2021 13:18:32] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:19:58,892 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:21:21,970 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:21:23,974 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    return self.finalize_request(rv)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1535, in finalize_request
    response = self.make_response(rv)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1727, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The return type must be a string, dict, tuple, Response instance, or WSGI callable, but it was a Client.
2021-05-30 13:21:23,976 INFO 127.0.0.1 - - [30/May/2021 13:21:23] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:22:46,118 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:22:49,320 INFO 127.0.0.1 - - [30/May/2021 13:22:49] "POST /clients HTTP/1.1" 200 -
2021-05-30 13:24:03,606 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:24:07,564 INFO 127.0.0.1 - - [30/May/2021 13:24:07] "POST /clients HTTP/1.1" 200 -
2021-05-30 13:25:16,955 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:25:19,616 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 26, in create_client
    client_service.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 14, in create_client
    return self.client_dao.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 13, in create_client
    cursor.execute(sql, client.client_name)
TypeError: not all arguments converted during string formatting
2021-05-30 13:25:19,619 INFO 127.0.0.1 - - [30/May/2021 13:25:19] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:31:10,238 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:31:23,877 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:31:28,766 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 26, in create_client
    client_service.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 14, in create_client
    return self.client_dao.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 13, in create_client
    cursor.execute(sql)
psycopg2.errors.UndefinedColumn: column "zack" does not exist
LINE 1: insert into client (client_name) values (Zack) returning cli...
                                                 ^

2021-05-30 13:31:28,770 INFO 127.0.0.1 - - [30/May/2021 13:31:28] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:32:25,856 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:32:29,390 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 26, in create_client
    client_service.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 14, in create_client
    return self.client_dao.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 13, in create_client
    cursor.execute(sql)
psycopg2.errors.UndefinedColumn: column "Zack" does not exist
LINE 1: insert into client (client_name) values ("Zack") returning c...
                                                 ^

2021-05-30 13:32:29,393 INFO 127.0.0.1 - - [30/May/2021 13:32:29] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-30 13:34:24,582 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 13:34:28,252 INFO 127.0.0.1 - - [30/May/2021 13:34:28] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 13:37:20,190 INFO 127.0.0.1 - - [30/May/2021 13:37:20] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 13:38:54,831 INFO 127.0.0.1 - - [30/May/2021 13:38:54] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 13:38:58,745 INFO 127.0.0.1 - - [30/May/2021 13:38:58] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 13:39:01,086 INFO 127.0.0.1 - - [30/May/2021 13:39:01] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 13:39:03,082 INFO 127.0.0.1 - - [30/May/2021 13:39:03] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 16:06:56,226 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 16:07:01,379 INFO 127.0.0.1 - - [30/May/2021 16:07:01] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 16:07:05,645 INFO 127.0.0.1 - - [30/May/2021 16:07:05] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 16:07:10,850 INFO 127.0.0.1 - - [30/May/2021 16:07:10] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 16:07:15,133 INFO 127.0.0.1 - - [30/May/2021 16:07:15] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 16:07:17,440 INFO 127.0.0.1 - - [30/May/2021 16:07:17] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 16:07:19,943 INFO 127.0.0.1 - - [30/May/2021 16:07:19] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 17:37:55,064 INFO 127.0.0.1 - - [30/May/2021 17:37:55] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 17:44:13,112 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 17:55:20,699 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 17:55:26,719 INFO 127.0.0.1 - - [30/May/2021 17:55:26] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 17:55:29,094 INFO 127.0.0.1 - - [30/May/2021 17:55:29] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 17:57:58,096 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 18:00:34,292 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 18:01:26,501 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 18:07:03,854 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 18:07:07,434 INFO 127.0.0.1 - - [30/May/2021 18:07:07] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 18:07:09,941 INFO 127.0.0.1 - - [30/May/2021 18:07:09] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 18:50:15,799 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 18:50:20,263 INFO 127.0.0.1 - - [30/May/2021 18:50:20] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 18:50:21,986 INFO 127.0.0.1 - - [30/May/2021 18:50:21] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 18:56:39,493 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 18:56:44,083 INFO 127.0.0.1 - - [30/May/2021 18:56:44] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 18:59:13,959 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 19:00:09,687 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 19:01:04,338 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 19:01:28,253 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 19:06:15,439 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 19:06:19,914 INFO 127.0.0.1 - - [30/May/2021 19:06:19] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 19:06:22,951 INFO 127.0.0.1 - - [30/May/2021 19:06:22] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 19:06:24,810 INFO 127.0.0.1 - - [30/May/2021 19:06:24] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 19:06:26,246 INFO 127.0.0.1 - - [30/May/2021 19:06:26] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 19:13:23,869 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 19:13:32,728 INFO 127.0.0.1 - - [30/May/2021 19:13:32] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 19:27:22,943 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 19:27:26,889 INFO 127.0.0.1 - - [30/May/2021 19:27:26] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 19:27:35,049 INFO 127.0.0.1 - - [30/May/2021 19:27:35] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 19:27:51,198 INFO 127.0.0.1 - - [30/May/2021 19:27:51] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 19:28:34,200 INFO 127.0.0.1 - - [30/May/2021 19:28:34] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 19:28:49,429 INFO 127.0.0.1 - - [30/May/2021 19:28:49] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 19:53:02,739 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 19:59:44,901 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 20:00:44,704 INFO 127.0.0.1 - - [30/May/2021 20:00:44] "[35m[1mPOST /clients/1/accounts HTTP/1.1[0m" 201 -
2021-05-30 20:03:00,114 INFO 127.0.0.1 - - [30/May/2021 20:03:00] "[35m[1mPOST /clients/1/accounts HTTP/1.1[0m" 201 -
2021-05-30 20:05:24,154 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 20:05:27,531 INFO 127.0.0.1 - - [30/May/2021 20:05:27] "[35m[1mPOST /clients/2/accounts HTTP/1.1[0m" 201 -
2021-05-30 20:05:45,726 INFO 127.0.0.1 - - [30/May/2021 20:05:45] "[35m[1mPOST /clients/2/accounts HTTP/1.1[0m" 201 -
2021-05-30 20:05:47,240 INFO 127.0.0.1 - - [30/May/2021 20:05:47] "[35m[1mPOST /clients/2/accounts HTTP/1.1[0m" 201 -
2021-05-30 20:45:55,941 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 20:46:04,343 INFO 127.0.0.1 - - [30/May/2021 20:46:04] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 20:46:08,074 INFO 127.0.0.1 - - [30/May/2021 20:46:08] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-30 20:46:16,225 INFO 127.0.0.1 - - [30/May/2021 20:46:16] "[35m[1mPOST /clients/2/accounts HTTP/1.1[0m" 201 -
2021-05-30 20:46:19,948 INFO 127.0.0.1 - - [30/May/2021 20:46:19] "[35m[1mPOST /clients/2/accounts HTTP/1.1[0m" 201 -
2021-05-30 20:48:25,728 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 20:57:42,696 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 20:58:06,669 INFO 127.0.0.1 - - [30/May/2021 20:58:06] "[33mGET /clients/ HTTP/1.1[0m" 404 -
2021-05-30 20:58:21,406 ERROR Exception on /clients [GET]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 41, in show_all_clients
    clients = client_service.show_all_clients()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 17, in show_all_clients
    return self.client_dao.show_all_clients()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 25, in show_all_clients
    client_list.append(Client(*x))
TypeError: __init__() takes 3 positional arguments but 7 were given
2021-05-30 20:58:21,441 INFO 127.0.0.1 - - [30/May/2021 20:58:21] "[35m[1mGET /clients HTTP/1.1[0m" 500 -
2021-05-30 20:59:43,431 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 20:59:50,121 INFO 127.0.0.1 - - [30/May/2021 20:59:50] "GET /clients HTTP/1.1" 200 -
2021-05-30 21:04:38,963 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:04:55,818 ERROR Exception on /clients/4 [GET]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 49, in show_client_by_id
    return jsonify(client.as_json_dict())
AttributeError: 'list' object has no attribute 'as_json_dict'
2021-05-30 21:04:55,821 INFO 127.0.0.1 - - [30/May/2021 21:04:55] "[35m[1mGET /clients/4 HTTP/1.1[0m" 500 -
2021-05-30 21:07:58,541 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:08:01,731 ERROR Exception on /clients/4 [GET]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 49, in show_client_by_id
    json_client = [c.as_json_dict() for c in client]
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 49, in <listcomp>
    json_client = [c.as_json_dict() for c in client]
AttributeError: 'tuple' object has no attribute 'as_json_dict'
2021-05-30 21:08:01,734 INFO 127.0.0.1 - - [30/May/2021 21:08:01] "[35m[1mGET /clients/4 HTTP/1.1[0m" 500 -
2021-05-30 21:09:16,505 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:09:19,276 ERROR Exception on /clients/4 [GET]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 49, in show_client_by_id
    return jsonify(client.as_json_dict())
AttributeError: 'list' object has no attribute 'as_json_dict'
2021-05-30 21:09:19,278 INFO 127.0.0.1 - - [30/May/2021 21:09:19] "[35m[1mGET /clients/4 HTTP/1.1[0m" 500 -
2021-05-30 21:12:05,352 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:12:14,117 ERROR Exception on /clients/4 [GET]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 48, in show_client_by_id
    client = client_service.show_client_by_id(int(client_id))
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 20, in show_client_by_id
    return self.client_dao.show_client_by_id(client_id)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 34, in show_client_by_id
    client_list.append(Client(client))
TypeError: __init__() missing 1 required positional argument: 'client_name'
2021-05-30 21:12:14,121 INFO 127.0.0.1 - - [30/May/2021 21:12:14] "[35m[1mGET /clients/4 HTTP/1.1[0m" 500 -
2021-05-30 21:16:48,472 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:16:53,501 INFO 127.0.0.1 - - [30/May/2021 21:16:53] "GET /clients/4 HTTP/1.1" 200 -
2021-05-30 21:17:00,838 INFO 127.0.0.1 - - [30/May/2021 21:17:00] "GET /clients/8 HTTP/1.1" 200 -
2021-05-30 21:17:04,884 INFO 127.0.0.1 - - [30/May/2021 21:17:04] "GET /clients/12 HTTP/1.1" 200 -
2021-05-30 21:23:35,812 INFO 127.0.0.1 - - [30/May/2021 21:23:35] "[35m[1mPOST /clients/2/accounts HTTP/1.1[0m" 201 -
2021-05-30 21:24:29,769 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:24:35,035 INFO 127.0.0.1 - - [30/May/2021 21:24:35] "GET /clients/2/accounts HTTP/1.1" 200 -
2021-05-30 21:32:30,583 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:35:15,880 INFO 127.0.0.1 - - [30/May/2021 21:35:15] "GET /clients/2/accounts/12 HTTP/1.1" 200 -
2021-05-30 21:35:24,417 INFO 127.0.0.1 - - [30/May/2021 21:35:24] "GET /clients/2/accounts/13 HTTP/1.1" 200 -
2021-05-30 21:35:31,741 INFO 127.0.0.1 - - [30/May/2021 21:35:31] "GET /clients/2/accounts HTTP/1.1" 200 -
2021-05-30 21:35:42,451 INFO 127.0.0.1 - - [30/May/2021 21:35:42] "GET /clients/2/accounts/1 HTTP/1.1" 200 -
2021-05-30 21:35:48,755 INFO 127.0.0.1 - - [30/May/2021 21:35:48] "GET /clients/2/accounts HTTP/1.1" 200 -
2021-05-30 21:41:04,774 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:43:01,013 INFO 127.0.0.1 - - [30/May/2021 21:43:01] "GET /clients/2/accounts?amountLessThan=300&amountGreaterThan=700 HTTP/1.1" 200 -
2021-05-30 21:47:27,037 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:47:30,742 INFO 127.0.0.1 - - [30/May/2021 21:47:30] "GET /clients/2/accounts?amountLessThan=300&amountGreaterThan=700 HTTP/1.1" 200 -
2021-05-30 21:48:37,063 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:48:42,474 INFO 127.0.0.1 - - [30/May/2021 21:48:42] "GET /clients/2/accounts?amountLessThan=300&amountGreaterThan=700 HTTP/1.1" 200 -
2021-05-30 21:49:19,839 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:49:26,137 INFO 127.0.0.1 - - [30/May/2021 21:49:26] "GET /clients/2/accounts?amountLessThan=300&amountGreaterThan=700 HTTP/1.1" 200 -
2021-05-30 21:50:55,494 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:50:59,280 INFO 127.0.0.1 - - [30/May/2021 21:50:59] "GET /clients/2/accounts?amountLessThan=300&amountGreaterThan=700 HTTP/1.1" 200 -
2021-05-30 21:54:41,095 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:54:45,367 INFO 127.0.0.1 - - [30/May/2021 21:54:45] "GET /clients/2/accounts?amountLessThan=300&amountGreaterThan=700 HTTP/1.1" 200 -
2021-05-30 21:56:54,469 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 21:57:02,156 INFO 127.0.0.1 - - [30/May/2021 21:57:02] "GET /clients/2/accounts?amountLessThan=300&amountGreaterThan=700 HTTP/1.1" 200 -
2021-05-30 22:00:13,146 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 22:00:17,338 INFO 127.0.0.1 - - [30/May/2021 22:00:17] "GET /clients/2/accounts?amountLessThan=300&amountGreaterThan=700 HTTP/1.1" 200 -
2021-05-30 22:09:08,889 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 22:09:14,279 INFO 127.0.0.1 - - [30/May/2021 22:09:14] "GET /clients/2/accounts?amountLessThan=300&amountGreaterThan=700 HTTP/1.1" 200 -
2021-05-30 22:13:47,810 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 22:13:53,289 INFO 127.0.0.1 - - [30/May/2021 22:13:53] "GET /clients/2/accounts?amountLessThan=300&amountGreaterThan=700 HTTP/1.1" 200 -
2021-05-30 22:23:21,580 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 22:28:11,180 INFO 127.0.0.1 - - [30/May/2021 22:28:11] "GET /clients/2/accounts?amountLessThan=300&amountGreaterThan=700 HTTP/1.1" 200 -
2021-05-30 22:41:10,327 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 22:42:35,429 INFO 127.0.0.1 - - [30/May/2021 22:42:35] "PUT /clients/2 HTTP/1.1" 200 -
2021-05-30 22:43:55,762 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 22:44:01,655 INFO 127.0.0.1 - - [30/May/2021 22:44:01] "PUT /clients/2 HTTP/1.1" 200 -
2021-05-30 22:44:11,889 INFO 127.0.0.1 - - [30/May/2021 22:44:11] "PUT /clients/2 HTTP/1.1" 200 -
2021-05-30 22:53:53,622 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-30 22:55:20,702 INFO 127.0.0.1 - - [30/May/2021 22:55:20] "PUT /clients/2/accounts/1 HTTP/1.1" 200 -
2021-05-30 22:59:32,891 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 13:51:10,757 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 13:51:17,053 INFO 127.0.0.1 - - [31/May/2021 13:51:17] "GET /clients HTTP/1.1" 200 -
2021-05-31 13:51:39,417 INFO 127.0.0.1 - - [31/May/2021 13:51:39] "GET /clients/2 HTTP/1.1" 200 -
2021-05-31 13:51:43,514 INFO 127.0.0.1 - - [31/May/2021 13:51:43] "GET /clients/1 HTTP/1.1" 200 -
2021-05-31 13:51:48,536 INFO 127.0.0.1 - - [31/May/2021 13:51:48] "GET /clients/4 HTTP/1.1" 200 -
2021-05-31 13:51:56,417 INFO 127.0.0.1 - - [31/May/2021 13:51:56] "GET /clients/2/accounts HTTP/1.1" 200 -
2021-05-31 13:52:08,610 INFO 127.0.0.1 - - [31/May/2021 13:52:08] "GET /clients/2/accounts/1 HTTP/1.1" 200 -
2021-05-31 13:52:19,940 INFO 127.0.0.1 - - [31/May/2021 13:52:19] "GET /clients/2/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-05-31 13:54:52,503 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 13:54:58,445 INFO 127.0.0.1 - - [31/May/2021 13:54:58] "GET /clients/2/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-05-31 14:17:24,875 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 14:17:37,060 ERROR Exception on /clients/2/accounts/1 [PATCH]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 96, in add_remove_funds
    account_service.add_remove_funds(int(client_id), int(account_id), change)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\account_service_impl.py", line 29, in add_remove_funds
    return self.account_dao.add_remove_funds(client_id, account_id, change)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\account_dao_impl.py", line 70, in add_remove_funds
    new_amount = int(account.funds) - change
AttributeError: 'tuple' object has no attribute 'funds'
2021-05-31 14:17:37,112 INFO 127.0.0.1 - - [31/May/2021 14:17:37] "[35m[1mPATCH /clients/2/accounts/1 HTTP/1.1[0m" 500 -
2021-05-31 14:19:28,100 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 14:19:31,052 ERROR Exception on /clients/2/accounts/1 [PATCH]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 95, in add_remove_funds
    change: int = body[0]
KeyError: 0
2021-05-31 14:19:31,055 INFO 127.0.0.1 - - [31/May/2021 14:19:31] "[35m[1mPATCH /clients/2/accounts/1 HTTP/1.1[0m" 500 -
2021-05-31 14:21:01,365 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 14:21:06,283 INFO 127.0.0.1 - - [31/May/2021 14:21:06] "PATCH /clients/2/accounts/1 HTTP/1.1" 200 -
2021-05-31 14:21:37,669 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 14:21:40,918 ERROR Exception on /clients/2/accounts/1 [PATCH]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 96, in add_remove_funds
    account_service.add_remove_funds(int(client_id), int(account_id), change)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\account_service_impl.py", line 29, in add_remove_funds
    return self.account_dao.add_remove_funds(client_id, account_id, change)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\account_dao_impl.py", line 70, in add_remove_funds
    new_amount = int(account.funds) - change
AttributeError: 'tuple' object has no attribute 'funds'
2021-05-31 14:21:40,921 INFO 127.0.0.1 - - [31/May/2021 14:21:40] "[35m[1mPATCH /clients/2/accounts/1 HTTP/1.1[0m" 500 -
2021-05-31 14:23:32,293 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 14:23:36,472 INFO 127.0.0.1 - - [31/May/2021 14:23:36] "PATCH /clients/2/accounts/1 HTTP/1.1" 200 -
2021-05-31 14:25:00,176 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 14:25:07,239 INFO 127.0.0.1 - - [31/May/2021 14:25:07] "PATCH /clients/2/accounts/1 HTTP/1.1" 200 -
2021-05-31 14:39:36,327 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 14:40:13,916 ERROR Exception on /clients/2/accounts/1 [PATCH]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 95, in add_remove_funds
    if body["withdraw"]:
KeyError: 'withdraw'
2021-05-31 14:40:13,919 INFO 127.0.0.1 - - [31/May/2021 14:40:13] "[35m[1mPATCH /clients/2/accounts/1 HTTP/1.1[0m" 500 -
2021-05-31 14:43:05,555 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 14:43:15,249 INFO 127.0.0.1 - - [31/May/2021 14:43:15] "PATCH /clients/2/accounts/1 HTTP/1.1" 200 -
2021-05-31 14:44:22,542 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 14:44:30,872 INFO 127.0.0.1 - - [31/May/2021 14:44:30] "PATCH /clients/2/accounts/1 HTTP/1.1" 200 -
2021-05-31 14:44:39,621 INFO 127.0.0.1 - - [31/May/2021 14:44:39] "PATCH /clients/2/accounts/1 HTTP/1.1" 200 -
2021-05-31 14:45:00,776 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 14:45:13,592 INFO 127.0.0.1 - - [31/May/2021 14:45:13] "[31m[1mPATCH /clients/2/accounts/1 HTTP/1.1[0m" 422 -
2021-05-31 14:45:33,609 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 15:02:00,997 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 15:02:27,708 INFO 127.0.0.1 - - [31/May/2021 15:02:27] "PATCH /clients/2/accounts/2/transfer/1 HTTP/1.1" 200 -
2021-05-31 15:08:11,993 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 15:12:46,087 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 15:13:46,782 INFO 127.0.0.1 - - [31/May/2021 15:13:46] "[35m[1mDELETE /clients/3 HTTP/1.1[0m" 201 -
2021-05-31 15:14:01,123 INFO 127.0.0.1 - - [31/May/2021 15:14:01] "[35m[1mDELETE /clients/2 HTTP/1.1[0m" 201 -
2021-05-31 15:17:01,330 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 15:17:23,657 INFO 127.0.0.1 - - [31/May/2021 15:17:23] "[35m[1mDELETE /clients/1/accounts/2 HTTP/1.1[0m" 201 -
2021-05-31 15:18:42,765 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 18:14:00,870 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 18:14:11,167 INFO 127.0.0.1 - - [31/May/2021 18:14:11] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-31 18:14:13,956 INFO 127.0.0.1 - - [31/May/2021 18:14:13] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-31 18:14:21,999 INFO 127.0.0.1 - - [31/May/2021 18:14:21] "[35m[1mPOST /clients/30/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:14:23,782 INFO 127.0.0.1 - - [31/May/2021 18:14:23] "[35m[1mPOST /clients/30/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:14:25,229 INFO 127.0.0.1 - - [31/May/2021 18:14:25] "[35m[1mPOST /clients/30/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:14:32,468 INFO 127.0.0.1 - - [31/May/2021 18:14:32] "GET /clients HTTP/1.1" 200 -
2021-05-31 18:14:46,023 INFO 127.0.0.1 - - [31/May/2021 18:14:46] "GET /clients/30 HTTP/1.1" 200 -
2021-05-31 18:14:54,814 INFO 127.0.0.1 - - [31/May/2021 18:14:54] "GET /clients/30/accounts HTTP/1.1" 200 -
2021-05-31 18:15:06,383 INFO 127.0.0.1 - - [31/May/2021 18:15:06] "GET /clients/2/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:15:10,823 INFO 127.0.0.1 - - [31/May/2021 18:15:10] "GET /clients/2/accounts/2 HTTP/1.1" 200 -
2021-05-31 18:15:14,859 INFO 127.0.0.1 - - [31/May/2021 18:15:14] "GET /clients/2/accounts/3 HTTP/1.1" 200 -
2021-05-31 18:15:29,612 INFO 127.0.0.1 - - [31/May/2021 18:15:29] "GET /clients/30/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-05-31 18:16:00,833 INFO 127.0.0.1 - - [31/May/2021 18:16:00] "PUT /clients/30 HTTP/1.1" 200 -
2021-05-31 18:16:06,194 INFO 127.0.0.1 - - [31/May/2021 18:16:06] "GET /clients/30 HTTP/1.1" 200 -
2021-05-31 18:16:26,947 INFO 127.0.0.1 - - [31/May/2021 18:16:26] "PUT /clients/30/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:16:37,277 INFO 127.0.0.1 - - [31/May/2021 18:16:37] "GET /clients/30/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:16:54,054 INFO 127.0.0.1 - - [31/May/2021 18:16:54] "PATCH /clients/30/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:17:01,127 INFO 127.0.0.1 - - [31/May/2021 18:17:01] "GET /clients/30/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:17:08,578 INFO 127.0.0.1 - - [31/May/2021 18:17:08] "PATCH /clients/30/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:17:13,763 INFO 127.0.0.1 - - [31/May/2021 18:17:13] "GET /clients/30/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:17:30,758 INFO 127.0.0.1 - - [31/May/2021 18:17:30] "GET /clients/30/accounts HTTP/1.1" 200 -
2021-05-31 18:17:52,289 INFO 127.0.0.1 - - [31/May/2021 18:17:52] "[35m[1mPOST /clients/30/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:17:59,780 INFO 127.0.0.1 - - [31/May/2021 18:17:59] "GET /clients/30/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:18:04,510 INFO 127.0.0.1 - - [31/May/2021 18:18:04] "GET /clients/30/accounts HTTP/1.1" 200 -
2021-05-31 18:21:56,056 INFO 127.0.0.1 - - [31/May/2021 18:21:56] "GET /clients HTTP/1.1" 200 -
2021-05-31 18:25:58,544 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 18:26:03,828 INFO 127.0.0.1 - - [31/May/2021 18:26:03] "GET /clients HTTP/1.1" 200 -
2021-05-31 18:26:18,440 INFO 127.0.0.1 - - [31/May/2021 18:26:18] "GET /clients/30/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-05-31 18:27:23,156 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 18:27:33,380 INFO 127.0.0.1 - - [31/May/2021 18:27:33] "GET /clients/2/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-05-31 18:28:57,152 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 18:29:01,421 INFO 127.0.0.1 - - [31/May/2021 18:29:01] "GET /clients/2/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-05-31 18:32:01,572 ERROR Exception on /clients/accounts [GET]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 48, in show_client_by_id
    client = client_service.show_client_by_id(int(client_id))
ValueError: invalid literal for int() with base 10: 'accounts'
2021-05-31 18:32:01,575 INFO 127.0.0.1 - - [31/May/2021 18:32:01] "[35m[1mGET /clients/accounts HTTP/1.1[0m" 500 -
2021-05-31 18:32:09,080 INFO 127.0.0.1 - - [31/May/2021 18:32:09] "GET /clients/2/accounts HTTP/1.1" 200 -
2021-05-31 18:33:07,145 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 18:33:48,836 INFO 127.0.0.1 - - [31/May/2021 18:33:48] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-31 18:33:55,186 ERROR Exception on /clients/30/accounts [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 35, in create_account
    account_service.create_account(account)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\account_service_impl.py", line 14, in create_account
    return self.account_dao.create_account(account)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\account_dao_impl.py", line 24, in create_account
    cursor.execute(sql)
psycopg2.errors.ForeignKeyViolation: insert or update on table "account" violates foreign key constraint "fk_account_client"
DETAIL:  Key (c_id)=(30) is not present in table "client".

2021-05-31 18:33:55,189 INFO 127.0.0.1 - - [31/May/2021 18:33:55] "[35m[1mPOST /clients/30/accounts HTTP/1.1[0m" 500 -
2021-05-31 18:34:03,580 ERROR Exception on /clients/1/accounts [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 35, in create_account
    account_service.create_account(account)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\account_service_impl.py", line 14, in create_account
    return self.account_dao.create_account(account)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\account_dao_impl.py", line 12, in create_account
    cursor.execute(sql)
psycopg2.errors.InFailedSqlTransaction: current transaction is aborted, commands ignored until end of transaction block

2021-05-31 18:34:03,581 INFO 127.0.0.1 - - [31/May/2021 18:34:03] "[35m[1mPOST /clients/1/accounts HTTP/1.1[0m" 500 -
2021-05-31 18:34:40,569 ERROR Exception on /clients/1/accounts [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 35, in create_account
    account_service.create_account(account)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\account_service_impl.py", line 14, in create_account
    return self.account_dao.create_account(account)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\account_dao_impl.py", line 12, in create_account
    cursor.execute(sql)
psycopg2.errors.InFailedSqlTransaction: current transaction is aborted, commands ignored until end of transaction block

2021-05-31 18:34:40,570 INFO 127.0.0.1 - - [31/May/2021 18:34:40] "[35m[1mPOST /clients/1/accounts HTTP/1.1[0m" 500 -
2021-05-31 18:36:19,647 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 18:36:32,509 INFO 127.0.0.1 - - [31/May/2021 18:36:32] "[35m[1mPOST /clients/1/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:36:39,131 INFO 127.0.0.1 - - [31/May/2021 18:36:39] "[35m[1mPOST /clients/1/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:36:40,600 INFO 127.0.0.1 - - [31/May/2021 18:36:40] "[35m[1mPOST /clients/1/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:37:21,576 INFO 127.0.0.1 - - [31/May/2021 18:37:21] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-31 18:37:28,787 INFO 127.0.0.1 - - [31/May/2021 18:37:28] "[35m[1mPOST /clients/2/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:37:30,183 INFO 127.0.0.1 - - [31/May/2021 18:37:30] "[35m[1mPOST /clients/2/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:37:31,493 INFO 127.0.0.1 - - [31/May/2021 18:37:31] "[35m[1mPOST /clients/2/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:37:49,669 INFO 127.0.0.1 - - [31/May/2021 18:37:49] "[35m[1mDELETE /clients/1/accounts/2 HTTP/1.1[0m" 201 -
2021-05-31 18:38:00,788 INFO 127.0.0.1 - - [31/May/2021 18:38:00] "[35m[1mPOST /clients/2/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:38:28,333 INFO 127.0.0.1 - - [31/May/2021 18:38:28] "[35m[1mDELETE /clients/2/accounts/2 HTTP/1.1[0m" 201 -
2021-05-31 18:39:31,506 INFO 127.0.0.1 - - [31/May/2021 18:39:31] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-31 18:40:44,596 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 18:40:53,586 INFO 127.0.0.1 - - [31/May/2021 18:40:53] "GET /clients/3/accounts HTTP/1.1" 200 -
2021-05-31 18:40:59,347 INFO 127.0.0.1 - - [31/May/2021 18:40:59] "GET /clients/2/accounts HTTP/1.1" 200 -
2021-05-31 18:42:30,772 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 18:42:44,369 INFO 127.0.0.1 - - [31/May/2021 18:42:44] "[35m[1mPOST /clients/3/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:42:46,121 INFO 127.0.0.1 - - [31/May/2021 18:42:46] "[35m[1mPOST /clients/3/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:42:47,559 INFO 127.0.0.1 - - [31/May/2021 18:42:47] "[35m[1mPOST /clients/3/accounts HTTP/1.1[0m" 201 -
2021-05-31 18:42:53,742 INFO 127.0.0.1 - - [31/May/2021 18:42:53] "GET /clients/3/accounts HTTP/1.1" 200 -
2021-05-31 18:43:02,121 INFO 127.0.0.1 - - [31/May/2021 18:43:02] "GET /clients/3/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:43:10,013 INFO 127.0.0.1 - - [31/May/2021 18:43:10] "GET /clients/3/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-05-31 18:44:08,499 INFO 127.0.0.1 - - [31/May/2021 18:44:08] "PUT /clients/3 HTTP/1.1" 200 -
2021-05-31 18:44:14,751 INFO 127.0.0.1 - - [31/May/2021 18:44:14] "GET /clients/3 HTTP/1.1" 200 -
2021-05-31 18:44:23,995 INFO 127.0.0.1 - - [31/May/2021 18:44:23] "PUT /clients/3/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:44:30,234 INFO 127.0.0.1 - - [31/May/2021 18:44:30] "GET /clients/3/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:44:46,906 INFO 127.0.0.1 - - [31/May/2021 18:44:46] "PATCH /clients/3/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:44:50,961 INFO 127.0.0.1 - - [31/May/2021 18:44:50] "GET /clients/3/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:44:56,213 INFO 127.0.0.1 - - [31/May/2021 18:44:56] "PATCH /clients/3/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:45:00,910 INFO 127.0.0.1 - - [31/May/2021 18:45:00] "GET /clients/3/accounts/1 HTTP/1.1" 200 -
2021-05-31 18:45:11,990 INFO 127.0.0.1 - - [31/May/2021 18:45:11] "PATCH /clients/3/accounts/1/transfer/2 HTTP/1.1" 200 -
2021-05-31 18:45:18,973 INFO 127.0.0.1 - - [31/May/2021 18:45:18] "GET /clients/3/accounts HTTP/1.1" 200 -
2021-05-31 18:45:47,901 INFO 127.0.0.1 - - [31/May/2021 18:45:47] "[35m[1mDELETE /clients/3 HTTP/1.1[0m" 201 -
2021-05-31 18:46:46,477 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 18:47:15,414 INFO 127.0.0.1 - - [31/May/2021 18:47:15] "GET /clients HTTP/1.1" 200 -
2021-05-31 19:09:15,135 ERROR Exception on /clients [POST]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 26, in create_client
    client_service.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\client_service_impl.py", line 14, in create_client
    return self.client_dao.create_client(client)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\client_dao_impl.py", line 12, in create_client
    cursor.execute(sql)
psycopg2.errors.AdminShutdown: terminating connection due to administrator command
SSL connection has been closed unexpectedly

2021-05-31 19:09:15,167 INFO 127.0.0.1 - - [31/May/2021 19:09:15] "[35m[1mPOST /clients HTTP/1.1[0m" 500 -
2021-05-31 19:10:14,557 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-05-31 19:10:19,045 INFO 127.0.0.1 - - [31/May/2021 19:10:19] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-31 19:10:23,076 INFO 127.0.0.1 - - [31/May/2021 19:10:23] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-05-31 19:10:29,057 INFO 127.0.0.1 - - [31/May/2021 19:10:29] "[35m[1mPOST /clients/38/accounts HTTP/1.1[0m" 201 -
2021-05-31 19:10:32,409 INFO 127.0.0.1 - - [31/May/2021 19:10:32] "[35m[1mPOST /clients/38/accounts HTTP/1.1[0m" 201 -
2021-05-31 19:10:33,709 INFO 127.0.0.1 - - [31/May/2021 19:10:33] "[35m[1mPOST /clients/38/accounts HTTP/1.1[0m" 201 -
2021-05-31 19:10:35,421 INFO 127.0.0.1 - - [31/May/2021 19:10:35] "[35m[1mPOST /clients/38/accounts HTTP/1.1[0m" 201 -
2021-05-31 19:10:40,153 INFO 127.0.0.1 - - [31/May/2021 19:10:40] "GET /clients HTTP/1.1" 200 -
2021-05-31 19:10:48,785 INFO 127.0.0.1 - - [31/May/2021 19:10:48] "GET /clients/38 HTTP/1.1" 200 -
2021-05-31 19:10:53,762 INFO 127.0.0.1 - - [31/May/2021 19:10:53] "GET /clients/38/accounts HTTP/1.1" 200 -
2021-05-31 19:11:02,001 INFO 127.0.0.1 - - [31/May/2021 19:11:02] "GET /clients/37/accounts HTTP/1.1" 200 -
2021-05-31 19:11:06,783 INFO 127.0.0.1 - - [31/May/2021 19:11:06] "GET /clients/38/accounts HTTP/1.1" 200 -
2021-05-31 19:11:15,345 INFO 127.0.0.1 - - [31/May/2021 19:11:15] "GET /clients/38/accounts/1 HTTP/1.1" 200 -
2021-05-31 19:11:21,514 INFO 127.0.0.1 - - [31/May/2021 19:11:21] "GET /clients/38/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-05-31 19:11:31,947 INFO 127.0.0.1 - - [31/May/2021 19:11:31] "PUT /clients/38 HTTP/1.1" 200 -
2021-05-31 19:11:36,643 INFO 127.0.0.1 - - [31/May/2021 19:11:36] "GET /clients/38 HTTP/1.1" 200 -
2021-05-31 19:11:42,339 INFO 127.0.0.1 - - [31/May/2021 19:11:42] "PUT /clients/38/accounts/1 HTTP/1.1" 200 -
2021-05-31 19:11:45,715 INFO 127.0.0.1 - - [31/May/2021 19:11:45] "GET /clients/38/accounts/1 HTTP/1.1" 200 -
2021-05-31 19:12:01,998 INFO 127.0.0.1 - - [31/May/2021 19:12:01] "PATCH /clients/38/accounts/1 HTTP/1.1" 200 -
2021-05-31 19:12:11,104 INFO 127.0.0.1 - - [31/May/2021 19:12:11] "PATCH /clients/38/accounts/1 HTTP/1.1" 200 -
2021-05-31 19:12:16,537 INFO 127.0.0.1 - - [31/May/2021 19:12:16] "GET /clients/38/accounts/1 HTTP/1.1" 200 -
2021-05-31 19:12:22,733 ERROR Exception on /clients/3/accounts/1/transfer/2 [PATCH]
Traceback (most recent call last):
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\venv\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\main.py", line 114, in transfer_funds
    account_service.transfer_funds(int(client_id), int(account_id_1), int(account_id_2), int(amount))
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\services\account_service_impl.py", line 32, in transfer_funds
    return self.account_dao.transfer_funds(client_id, account_id_1, account_id_2, change)
  File "C:\Users\mrmas_d2t5g6y\Desktop\Programming\BankApp\daos\account_dao_impl.py", line 82, in transfer_funds
    first_change = int(first_account[3]) - change
TypeError: 'NoneType' object is not subscriptable
2021-05-31 19:12:22,736 INFO 127.0.0.1 - - [31/May/2021 19:12:22] "[35m[1mPATCH /clients/3/accounts/1/transfer/2 HTTP/1.1[0m" 500 -
2021-05-31 19:12:28,492 INFO 127.0.0.1 - - [31/May/2021 19:12:28] "PATCH /clients/38/accounts/1/transfer/2 HTTP/1.1" 200 -
2021-05-31 19:12:37,832 INFO 127.0.0.1 - - [31/May/2021 19:12:37] "GET /clients/38/accounts HTTP/1.1" 200 -
2021-05-31 19:12:55,047 INFO 127.0.0.1 - - [31/May/2021 19:12:55] "[35m[1mDELETE /clients/38/accounts/2 HTTP/1.1[0m" 201 -
2021-05-31 19:13:59,022 INFO 127.0.0.1 - - [31/May/2021 19:13:59] "[35m[1mDELETE /clients/38 HTTP/1.1[0m" 201 -
2021-05-31 19:15:56,864 INFO 127.0.0.1 - - [31/May/2021 19:15:56] "[35m[1mPOST /clients HTTP/1.1[0m" 201 -
2021-06-01 10:31:45,525 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-06-01 10:32:16,569 INFO 127.0.0.1 - - [01/Jun/2021 10:32:16] "PATCH /clients/1/accounts/1/transfer/2 HTTP/1.1" 200 -
2021-06-01 13:00:19,123 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-06-01 13:01:10,040 INFO 127.0.0.1 - - [01/Jun/2021 13:01:10] "GET /clients/2/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-06-01 13:05:05,012 INFO 127.0.0.1 - - [01/Jun/2021 13:05:05] "GET /clients/2/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-06-01 13:11:02,654 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-06-01 13:11:07,187 INFO 127.0.0.1 - - [01/Jun/2021 13:11:07] "GET /clients/2/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-06-01 13:13:39,832 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-06-01 13:13:47,368 INFO 127.0.0.1 - - [01/Jun/2021 13:13:47] "GET /clients/2/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
2021-06-01 13:14:56,411 INFO  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-06-01 13:15:03,207 INFO 127.0.0.1 - - [01/Jun/2021 13:15:03] "GET /clients/2/accounts?amountLessThan=950&amountGreaterThan=1050 HTTP/1.1" 200 -
